.github/workflows/labeler.yml

```
@@ -5,6 +5,8 @@ on:
 jobs:
   triage:
     runs-on: ubuntu-latest
+    permissions:
+        contents: write
```

```suggestion
    permissions:
        contents: read
        pull-requests: write
```
could we make the permissions more strict? 

additionally, it looks like a lot of projects have started using the offical labeler action instead of the fork, like integrations-core: https://github.com/DataDog/integrations-core/pull/12546. Maybe we could try this in a different PR?

---

I adjusted the permissions now, and yes we can do that move on a follow up PR.